library/spdm_responder_lib/libspdm_rsp_psk_exchange.c

```diff
@@ -359,6 +360,11 @@ return_status spdm_get_response_psk_exchange(IN void *context,
 
         spdm_set_session_state(spdm_context, session_id,
                        LIBSPDM_SESSION_STATE_ESTABLISHED);
+        result = start_watchdog(&spdm_context->watchdog_context, session_id,
```

Comment:
Heartbeats are only active in the application phase, so this should be in `PSK_FINISH_RSP`. `FINISH_RSP` needs it as well.